features.json

```diff
@@ -12,6 +12,7 @@
     "ember-container-inject-owner": true,
     "ember-htmlbars-local-lookup": null,
     "ember-application-engines": null,
-    "ember-glimmer": null
+    "ember-glimmer": null,
+    "ember-uniqBy": true
```

Comment:
Any way I can conditionally export this function without having this JSHint error fail the build? `ember-runtime/computed/reduce_computed_macros.js: line 394, col 1, Export declaration must be in global scope.`